storage/storage_tests/storage_tests.cpp

```diff
@@ -1158,6 +1158,40 @@ UNIT_TEST(StorageTest_ChildrenSizeSingleMwm)
   TEST_EQUAL(southKoreaCountry.GetSubtreeMwmSizeBytes(), 48394664, ());
 }
 
+UNIT_TEST(StorageTest_ParentSingleMwm)
+{
+  Storage storage(kSingleMwmCountriesTxt, make_unique<TestMapFilesDownloader>());
+
+  Country const abkhaziaCountry = storage.CountryByCountryId("Abkhazia");
+  TEST_EQUAL(abkhaziaCountry.GetParent(), "Countries", ());
+
+  Country const algeriaCentralCountry = storage.CountryByCountryId("Algeria_Central");
```

Comment:
Why not to use UNIT_CLASS_TEST and create a helper function like:
```c++
bool ParentOf(string const & parent, string const & country)
{
  Country c = storage.CountryByCountryId(country);
  return c.GetParent() == parent;
}
```
And use it here?